dynamic-security-analysis/src/test/resources/DynamicSecurityAnalysisParametersV11.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@thangqp  rename to DynamicSecurityAnalysisParameters.json
Lastest version don't have the version in the file name

Reply:
DONE in https://github.com/powsybl/powsybl-core/pull/3455/commits/253a3a153e5da77900995e4746a2936d0be3c321

DO the same for dynamic simulation